### PR TITLE
[FluidRelay] Generate fluid-relay with CodeGen v2

### PR DESCRIPTION
**Pull request by ZengTaoxu**
---
Closes: https://github.com/Azure/azure-cli/issues/22938

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

az fluid-relay server create
az fluid-relay server update
az fluid-relay server list
az fluid-relay server list-key
az fluid-relay server regenerate-key
az fluid-relay server show
az fluid-relay server delete
az fluid-relay container show
az fluid-relay container list
az fluid-relay container delete
### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
FluidRelay

File: src/fluid-relay/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Azure CLI FluidRelay Extension #
+This is an extension to Azure CLI to manage FluidRelay resources.
+
+## How to use ##
+Please add commands usage here.
```
Comment (necusjz) [inline src/fluid-relay/README.md:5]: It's better to have some basic use cases.

File: src/fluid-relay/azext_fluid_relay/aaz/latest/fluid_relay/container/_delete.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "fluid-relay container delete",
+    confirmation="Are you sure you want to perform this operation?",
+)
+class Delete(AAZCommand):
+    """Delete a Fluid Relay container.
```
Comment (necusjz) [inline src/fluid-relay/azext_fluid_relay/aaz/latest/fluid_relay/container/_delete.py:19]: It's better to complete the examples part, same as other commands.

File: src/service_name.json [hunk 3]
```diff
@@ -189,6 +189,11 @@
     "AzureServiceName": "Azure Elastic",
     "URL": "https://docs.microsoft.com/en-us/azure/partner-solutions/elastic/overview"
   },
+  {
+    "Command": "az fluid-relay",
+    "AzureServiceName": "Azure Fluid Relay",
+    "URL": "https://docs.microsoft.com/en-us/azure/azure-fluid-relay/overview"
```
Comment (necusjz) [inline src/service_name.json:195]: Page not found?

File: src/fluid-relay/azext_fluid_relay/azext_metadata.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+    "azext.isExperimental": true,
+    "azext.minCliCoreVersion": "2.38.0"
```
Comment (necusjz) [inline src/fluid-relay/azext_fluid_relay/azext_metadata.json:3]: Should wait for the release of Azure CLI 2.39.0
```suggestion
    "azext.minCliCoreVersion": "2.39.0"
```

Comment (ZengTaoxu) [reply src/fluid-relay/azext_fluid_relay/azext_metadata.json]: Thank you for your correction! Relevant questions have been revised.

File: src/fluid-relay/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
```
Comment (necusjz) [inline src/fluid-relay/README.md:5]: We can remove this hint line. :)

**Review by necusjz — APPROVED** [review]

**Pull request merged**